x/staging/posts/client/cli/tx.go

```diff
@@ -375,6 +385,7 @@ E.g.
 		},
 	}
 
+	cmd.Flags().StringSlice(FlagReason, []string{""}, "Reasons for the report")
```

Comment:
Can we simply specify an empty array here? 

```suggestion
	cmd.Flags().StringSlice(FlagReason, nil, "Reasons for the report")
```